lib/aggregate/attribute/bitfield.rb

```diff
@@ -42,6 +42,10 @@ def validation_errors(_value)
 
   private
 
+  def value_to_string(value)
+    value.is_a?(Array) ? value&.map { |v| klass.to_bit(v) }&.join : value&.to_s
```

Comment:
To be a bit more general with other enumerable types, maybe just check that it respond to `map`? And no need for `&` when it does:
```
value.respond_to?(:map) ? value.map { |v| klass.to_bit(v) }.join : value&.to_s
```

Reply:
Thanks, Colin! 👍

Reply:
Looks good! 👍